scripts/duplicate_end_to_end.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you merge this script into scripts/generate_duplicate_sheet.py (using a parameter to choose between using only the duplicate model or using similarity first then duplicate model)?